.env

```diff
@@ -77,3 +77,6 @@ FIFTYONE_HOST_URL=${FIFTYONE_HOST_IP}:${FIFTYONE_HOST_PORT}
 
 # viewer
 MONGODB_HOST_PORT=27019
+
+# algorithm store address, for front end
```

Comment:
# deploy module address, for web integration.

Reply:
fixed